guides/micronaut-cloud-secrets-azure/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,17 @@
+#tag::distributedconfiguration[]
+micronaut:
+  application:
+    name: micronautguide # <1>
+  config-client:
+    enabled: true # <2>
+#end::distributedconfiguration[]
+#tag::azure[]
+azure:
+  credential:
+    client-secret:
+      client-id: ${AZURE_CLIENT_ID} # <1>
```

Comment:
is this going to work given the Azure naming convention restriction for secrets, which does not allow underscores to be used?  Should it be `AZURE-CLIENT-ID`?